src/mail/mjml/invitation.mjml

```diff
@@ -0,0 +1,54 @@
+<mjml>
+  <mj-include path="./partial/header.mjml" />
+
+  <mj-body mj-class="bg--blue-100">
+    <mj-wrapper css-class="wrapper" padding="0 40px 40px 40px">
+      <mj-section background-url="{% base64_static 'images/mail-header-background.png' %}" background-size="cover" background-repeat="no-repeat" background-position="0 -30px">
```

Comment:
The images are missing: 
![image](https://github.com/numerique-gouv/impress/assets/25994652/b25eaa06-c157-46ab-9e99-708eeac63ae6)

You can get them here:
https://github.com/numerique-gouv/people/tree/main/src/backend/core/static/images